app/views/course/admin/announcement_settings/index.html.slim

```diff
@@ -0,0 +1,4 @@
+= simple_form_for @settings, url: course_admin_announcement_path(current_course) do |f|
```

Comment:
Then all exiting code need change ?  I actually prefer no brackets, and this also follows the style in official `simple_form` guide.  I know you have concern about DSL, but..